testing/integration_test.go

```diff
@@ -103,19 +105,31 @@ type nodeGCETestCase struct {
 
 func (tc *nodeGCETestCase) initializeStartUpScript(template *template.Template) error {
 	var buf bytes.Buffer
+
+	var checkoutBranchCmd string
+	if *pr != 0 {
+		checkoutBranchCmd = fmt.Sprintf(
+			`git fetch origin pull/%d/head:pull_branch
```

Comment:
This became a strange mix of using templates and formatting strings manually to my taste. I'd suggest passing both pr and branch to the template and use template conditions to format the commands the needed way. See "if" pipeline in https://golang.org/pkg/text/template/#hdr-Actions.

Reply:
Done.